Fix assign graders

andesol opened the pull request:
!! Only for debugging

## Done 

- Run Pupetteer in `headless: false` so the steps an be seen in the browser
- Add a bunch of console logs
- Add some validation around the config file: throw error if the file is empty or job name doesn't match the one in the `yml` file

## QA

- `yarn dev assign -i`
- Select job/s
- Let the process finish
- If there was any error let me know

anthonydillon commented:
It worked by assigning the four I had but then later hit the timeout. There were no extra logs.

anthonydillon commented:
Tricky, its working with this branch and but `ght` still gets the error so maybe its something to do with regular console logs to the terminal.

andesol commented:
@anthonydillon I removed any trace of debugging and changed the behaviour of the spinner so it shows "Processing applications" while going through the pages.

You can try again in this branch (warning: is going to assign people for real) or we can merge it and try with `ght`
